infra/debian/compiler/docs/one-infer.1

```diff
@@ -1,11 +1,11 @@
-.TH ONE-INFER "1" "June 2022" "one-profile version 1.21.0" "User Commands"
```

Comment:
I regenerated man page by `help2man` tool. There were few errors before, and it will be fixed from this patch